.clabot

```diff
@@ -47,6 +47,7 @@
     "VitaliiShpital",
     "isaachess",
     "azdagron",
-    "phthano"
+    "phthano",
+    "nerdatwork"
```

Comment:
Please remove this change. You need to be in the list before to let the check pass 👍

Reply:
Done.